elide-contrib/elide-test-helpers/src/test/java/com/yahoo/elide/contrib/testhelpers/jsonapi/JsonApiDSLTest.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+
+package com.yahoo.elide.contrib.testhelpers.jsonapi;
+
+import static com.yahoo.elide.contrib.testhelpers.jsonapi.JsonApiDSL.*;
```

Comment:
expand star import

Reply:
It is my sense that star imports are generally accepted for testing frameworks/libraries. 

http://hamcrest.org/JavaHamcrest/tutorial
http://joel-costigliola.github.io/assertj/

There would be a lot of imports for this DSL. Do we have a strong stance on this for this project.